pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -873,6 +871,22 @@ def start_neard(self, batch_interval_millis=None):
                     self.data['current_neard_path'], '--log-span-events',
                     '--home', self.neard_home, '--unsafe-fast-startup', 'run'
                 ]
+
+            # Save logs config file to control the level of rust and opentelemetry logs.
+            # Default config sets level to DEBUG for "client" and "chain" logs, WARN for tokio+actix, and INFO for everything else.
+            default_log_filter = 'client=debug,chain=debug,actix_web=warn,mio=warn,tokio_util=warn,actix_server=warn,actix_http=warn,info'
+            log_config_file_path = self.target_near_home_path(
```

Comment:
shouldn't this be `self.target_near_home_path()` whether or not it's the traffic generator? `self.home_path()` is a path in `~/.near/neard-runner` and `self.target_near_home_path()` is either `~/.near` or `~/.near/target` if it's a traffic generator

Reply:
oh right, I had the impression that it runs from the target directory. Changing.